sw/device/boot_rom/boot_rom.c

```diff
@@ -36,12 +36,12 @@ void _boot_start(void) {
 
   int bootstrap_err = bootstrap();
   if (bootstrap_err != 0) {
-    LOG_ERROR("Bootstrap failed with status code: %d\n", bootstrap_err);
+    base_printf("Bootstrap failed with status code: %d\r\n", bootstrap_err);
```

Comment:
Why change these?

Reply:
Odd. I think that's left over from something else I was doing. Good catch.